github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -767,7 +767,7 @@ periodics:
       args:
       - "-ce"
       - |
-        ./hack/git-pr.sh -c "bazel run //robots/cmd/dependabot:dependabot -- --github-token-path /etc/github/oauth --repo project-infra --org kubevirt --repo-dir $(pwd) && make gazelle gazelle-update-repos test" -b dependabot-update -T main
+        ./hack/git-pr.sh -c "bazel run //robots/cmd/dependabot:dependabot -- --github-token-path /etc/github/oauth --repo project-infra --org kubevirt --repo-dir $(pwd) && make gazelle gazelle-update-repos test" -b dependabot-update -r project-infra -T main
```

Comment:
@dhiller thinking about it, the `test` target should probably not be executed.

I think it is beneficial to increase the chance to create a PR with build failures or failing tests. Then the PR itself would show the issue (instead of a relatively invisible place), and it is easier to see that something needs to be changed. What do you think?